recommender/migrate/utils.py

```diff
@@ -0,0 +1,156 @@
+# pylint: disable=invalid-name
+
+"""Util methods for migration purposes"""
+
+import os
+from importlib import import_module
+
+from definitions import MIGRATE_DIR
+from recommender.models.migration import Migration
+
+
+def _get_migration_class(migration: Migration):
+    migration_name = migration.name
+    capitalized = [part.capitalize() for part in migration_name.split("_")[1:]]
+    class_name = "".join(capitalized)
+
+    migration_module = import_module(f"recommender.migrate.{migration_name}")
+    migration_class = getattr(migration_module, class_name)
+
+    return migration_class
+
+
+def _get_migrations_from_migrate_dir():
+    files = os.listdir(MIGRATE_DIR)
+    migration_names = [
+        file.split(".")[0] for file in files if file.split("_")[0].isnumeric()
+    ]
+    return migration_names
+
+
+def scan_for_migrations():
+    """
+    Scans for migrations in the MIGRATE_DIR and
+    then inserts newly found migrations into the Migration collection.
+    """
+
+    print("Scanning for new migrations...")
+
+    all_migration_names = _get_migrations_from_migrate_dir()
+    existing_migration_names = [migration.name for migration in Migration.objects()]
+    missing_migration_names = set(all_migration_names) - set(existing_migration_names)
+
+    if missing_migration_names:
+        for m in missing_migration_names:
```

Comment:
A little more descriptive variable name would be better - `migration_name` or something like that